modules/llm-cache/ds/kv_cache_manager.cc

```diff
@@ -21,13 +21,43 @@ limitations under the License.
 
 #include "client/client.h"
 #include "common/util/logging.h"
+#include "common/util/metrics.h"
 #include "common/util/status.h"
 #include "llm-cache/ds/kv_cache.h"
 #include "llm-cache/ds/kv_cache_manager.h"
 #include "llm-cache/storage/blob_storage.h"
 #include "llm-cache/storage/local_file_storage.h"
 #include "llm-cache/storage/vineyard_file_storage.h"
 
+#define KV_PREFIX "llm_kv_cache"
+#define KV_REQUEST_PREFIX KV_PREFIX "_request"
+
+#define LOG_UPDATE(op, start, end, status, prefix_size, curr_size,            \
+                   updated_size)                                              \
+  LOG_SUMMARY(KV_REQUEST_PREFIX "_nr_tokens", op "_prefix", prefix_size);     \
```

Comment:
any reason to use summary rather than histogram?

Reply:
Just follow the convention of v6d's metrics collection. We could switch to using histogram if needed. But if we only care about some particular percentiles, using summary is better than histogram w.r.t prometheus storage efficiency. We will lose the capability of calculating arbitrary percentiles though.

Reply:
I see you have some fixed percentile. `summary` makes sense here